service/controller/rbac.go

```diff
@@ -53,24 +70,26 @@ func NewTODO(config TODOConfig) (*TODO, error) {
 		}
 	}
 
-	c := &TODO{
+	c := &RBAC{
 		Controller: operatorkitController,
 	}
 
 	return c, nil
 }
 
-func newTODOResourceSets(config TODOConfig) ([]*controller.ResourceSet, error) {
+func newRBACResourceSets(config RBACConfig) ([]*controller.ResourceSet, error) {
```

Comment:
Let's get rid of this method. It's useless in a flat operator. Se my other suggestions.